truss/contexts/local_loader/local_server_loader.py

```diff
@@ -130,11 +128,7 @@ def _prep_or_create_dir(location: Optional[Path], stub: str) -> Path:
 
         TrussFilesSyncer(
             self.truss_path,
-            LocalTrussPatchApplier(
-                venv_dir / "app/",
-                venv_builder.context.env_exe,
-                logging.Logger(__name__),
-            ),
+            None,
```

Comment:
Does this mean that the local_loader no longer works? If so, you could just delete it all together. Doesn't make sense to keep it if it's not gonna work.

Reply:
I think that this still does work because of how we are running `uvicorn --reload` so it will reload the server but not with the patching mechanism. Happy to remove regardless if you think we should

Reply:
yeah. Let's drop it. It's not ready for primetime as much as the remote watch stuff, so let's double down on our strengths